pages/common/hg-status.md

```diff
@@ -0,0 +1,36 @@
+# hg status
+
+> Show files that have changed in the working directory.
+> Homepage: <https://www.mercurial-scm.org/doc/hg.1.html#status>.
+
+- Display the status of changed files:
+
+`hg status`
+
+- Display only modified files:
+
+`hg status --modified`
+
+- Display only added files:
+
+`hg status --added`
+
+- Display only removed files:
+
+`hg status --removed`
+
+- Display only deleted (but tracked) files:
+
+`hg status --deleted`
+
+- Display changes in the working directory compared to a specified changeset:
+
+`hg status --rev {{revision}}`
+
+- Display only files matching a specified pattern:
```

Comment:
Is this a glob or a regex?

Reply:
Its a glob e.g `*.md`

Reply:
Yeah, I think I mention that `hg` commands use globs in another `hg` command page. :thinking: